src/main.service.js

```diff
@@ -160,6 +160,28 @@ plugin.service('wgnConfigSrv', ['$q', '$firebase', 'znData', function ($q, $fire
 		return multi ? saveMulti(workspaceId, $ref, config) : saveSingle(workspaceId, $ref);
 	};
 
+	/**
+	 * Saves a plugin's secure configuration settings for the given workspace.
+	 *
+	 * @param {number} workspaceId
+	 * @param {String} configId The config id for multi configurations
+	 * @param {Object} config The configuration object to save.
+	 *
+	 * @return {Promise<*>}
+	 */
+	srv.saveSecure = function (workspaceId, configId, config) {
+
+		var params = {
+			workspaceId: workspaceId,
+			params: {
+				configId: configId
+			}
+		};
+
+		return znPluginData('wgn').post('/settings', params, config);
```

Comment:
This endpoint should probably be configurable with a default of `'/settings'`, just in case someone already has their backend service setup with a different name.

I'm thinking same interface as the other default properties:

```
new ConfigService('name')
  .title('title')
  .multi(true)
  .secure(true)
  .secureFieldEndpoint('/not-settings')
```